settlement-config.yaml

```diff
@@ -0,0 +1,24 @@
+---
+- LowCreditsSettlement:
+    meta:
+      funder: ValidatorBond
+    min_settlement_lamports: 100000000
+    grace_low_credits_bps: 100 # 1 %
+    covered_range_bps: [0, 2000] # 0 % - 20 %
```

Comment:
Let me explain on example:
Expected rewards 100 SOL, downtime 20 %, actual rewards will be 80 SOL.
The range [0, 20 %] covers at most 0.2 * 100 SOL, in this case 20 SOL.
The range [20%, 100%] would start covering if downtime was above 20 %. If it was 20 % exactly, it would have to cover exactly 0 SOL.

I am not sure it makes sense to talk about inclusion/exclusion in this case; If it does, then I think, strictly speaking, both bounds are included - but on the boundary, one of the settlements will be 0

Reply:
I see, thanks for the elaboration